tools/prepare_scene_for_editing.py

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+import argparse
+import os
+import spear
+
+
+if __name__ == '__main__':
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--perforce_content_dir", required=True)
+    parser.add_argument("--scene_id", required=True)
+    parser.add_argument("--remove_symlink", action="store_true")
+    args = parser.parse_args()
+    
+    assert os.path.exists(args.perforce_content_dir)
+
+    unreal_project_content_dir = os.path.realpath(os.path.join(os.path.dirname(__file__), "..", "cpp", "unreal_projects", "SpearSim", "Content"))
+
+    # Create and/or remove symlink for Shared directory
+    perforce_content_shared_dir = os.path.realpath(os.path.join(args.perforce_content_dir, "Shared"))
+
```

Comment:
To maintain consistency, it'd be good to 
```python
assert os.path.exists(perforce_content_shared_dir)
```
similar to 
```python
assert os.path.exists(perforce_content_scene_dir)
``` 
below.

Reply:
Fair point. Only one of them depends on an additional input argument that hasn't already been validated, so strictly speaking, only `perforce_content_scene_dir` really needs to be checked. But there is no harm in asserting for both, and I like the added consistency.